src/net/socket.rs

```diff
@@ -5,18 +5,17 @@ use crate::{
     packet::Packet,
 };
 use crossbeam_channel::{self, unbounded, Receiver, Sender};
-use log::error;
-use mio::{Evented, Events, Poll, PollOpt, Ready, Token};
+#[allow(unused_imports)]
```

Comment:
can we just remove the imports? 😄

Reply:
I'm fine with that, they are just there for logging purposes. I go back and forth between only importing the ones used, and importing them all to save future people the trouble. I dunno. I'll prolly just remove them.

Reply:
I actually have been prefering to just `macro_use` them since they are so common and are only a few macros that everyone knows.

Reply:
Example please?